octokit-project/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,22 @@
+package org.example;
+
+public class Main {
+
+    public static void main(String[] args) {
+        int a = 1;
+        float b = 2.0f;
+        double c = 3.0;
+        long d = 4L;
+        short e = 5;
```

Comment:
[Severity: Low] The use of 'System.out.println' for output is acceptable for simple demonstrations, but for larger applications, consider using a logging framework like SLF4J or Log4j.
This will allow for better control over logging levels and outputs.
Rationale: Using a logging framework enhances flexibility and maintainability of the code, especially in production environments.
Reference: The Java Logging Framework documentation provides guidelines on logging best practices.